rai/models.go

```diff
@@ -257,7 +257,7 @@ type TransactionAsyncFile struct {
 
 type ArrowRelation struct {
 	RelationID string
-	Table      []interface{}
+	Table      [][]interface{}
```

Comment:
You could maybe call this `Columns` if you want? But `Table` seems fine too. 👍

Reply:
Yeah, just to keep it aligned with other sdks :pray: